[Visual Refresh] Inputs: Add refresh design update #2

mgadewoll opened the pull request:
## Summary

>[!IMPORTANT]
This PR merges into a feature branch.
This PR should be rebased after the [first update PR](https://github.com/elastic/eui/pull/8767) is merged.

Implements https://github.com/elastic/eui-private/issues/324

This PR is the second part to implement Visual Refresh input design changes. This PR implements changes to specific components:
- `EuiComboBox`
- `EuiDatePicker`
- `EuiSuperDatePicker`
- `EuiFilePicker` ([figma](https://www.figma.com/design/FHys7gLzyvD1gc9DrJM6D8/Elastic-UI--Borealis-?node-id=9139-44435&m=dev))

### Changes

- updates padding on `EuiComboBox`
- prevents duplicate outlines in HCM for `EuiDatePicker`
- updates styles on `EuiSuperDatePicker` to ensure expected design output (applying/inheriting border radius and state colors)
- `EuiFilePicker`:
  - updates default border color
  - focus/hover/invalid state styles
  - updates drag-drop styling
  - icon color

## Why are we making this change?

This update align specialized form components with the updated base form styles (done [here](https://github.com/elastic/eui/pull/8767)) and applies new design updates. These include updates to text colors, border colors, state styles (focus, invalid, disabled, readonly), spacings and icon sizes. Additionally we're ensuring that the components work well in high contrast modes.

## Screenshots <a href="#user-content-screenshots" id="screenshots">#</a>

<details>
<summary>Toggle screenshots</summary>

#### LIGHT mode

##### EuiComboBox

| classic | refresh |
|---|---|
| ![Screenshot 2025-06-11 at 13 25 38](https://github.com/user-attachments/assets/b9e198c1-41b4-4dee-9562-9699d02ef72b) | ![Screenshot 2025-06-11 at 13 25 56](https://github.com/user-attachments/assets/05ec5069-51ab-4034-b150-b69cfba5e7f1) | 
| ![Screenshot 2025-06-11 at 13 30 04](https://github.com/user-attachments/assets/9ec500fb-d9ba-4778-8457-9cfecf37e568) | ![Screenshot 2025-06-11 at 13 29 32](https://github.com/user-attachments/assets/3fe0d507-ee9e-444a-a518-dc68861237f9) | 
| ![Screenshot 2025-06-11 at 13 30 09](https://github.com/user-attachments/assets/0a5f1643-6afc-4d8c-911a-e65c8c4425c4) | ![Screenshot 2025-06-11 at 13 29 37](https://github.com/user-attachments/assets/a3c95504-e49d-4106-9da3-19b2f4058462) | 

##### EuiDatePicker

| classic | refresh |
|---|---|
| ![Screenshot 2025-06-11 at 09 29 23](https://github.com/user-attachments/assets/ff11c527-a7ed-4c7a-a156-54ce02e156c8) | ![Screenshot 2025-06-11 at 09 29 44](https://github.com/user-attachments/assets/1387f400-b6f1-41c3-9af9-a47350514648) | 
| ![Screenshot 2025-06-11 at 09 31 20](https://github.com/user-attachments/assets/2d51d412-dd05-413e-99d9-af2196ced8e9) | ![Screenshot 2025-06-11 at 09 31 28](https://github.com/user-attachments/assets/069d559f-6e5b-4e9c-b0a9-1bc55c845c94) | 
| ![Screenshot 2025-06-11 at 09 33 49](https://github.com/user-attachments/assets/33419341-5f9e-47b0-a4c9-b44cd001f3c5) | ![Screenshot 2025-06-11 at 09 34 01](https://github.com/user-attachments/assets/c8285061-66cf-49eb-8a9e-872bced8d8d1) | 

##### EuiSuperDatePicker

| classic | refresh |
|---|---|
| ![Screenshot 2025-06-11 at 09 38 35](https://github.com/user-attachments/assets/b45730b3-a580-4cd8-a204-c51f5797ada5) | ![Screenshot 2025-06-11 at 09 39 58](https://github.com/user-attachments/assets/3b2f6d96-ec39-4155-87dc-d72833f4cb45) | 
| ![Screenshot 2025-06-11 at 09 48 07](https://github.com/user-attachments/assets/ac460456-d508-4ded-a9cc-6777aba88b8b) | ![Screenshot 2025-06-11 at 09 48 48](https://github.com/user-attachments/assets/474b5abe-c3ba-441b-887d-4dad3aa64ff3) | 
| ![Screenshot 2025-06-11 at 09 50 43](https://github.com/user-attachments/assets/7a3badbe-edd8-4046-abd0-2b3d8584ef82) | ![Screenshot 2025-06-11 at 09 51 23](https://github.com/user-attachments/assets/3934f426-9321-4ffb-963c-260f036b6942) | 
| ![Screenshot 2025-06-11 at 09 52 08](https://github.com/user-attachments/assets/0ba5c202-30f7-4d66-8ec8-7df15d84ff6a) | ![Screenshot 2025-06-11 at 09 53 15](https://github.com/user-attachments/assets/f12a23c4-b723-48dc-951f-0957dbc4acbc) | 

##### EuiFilePicker

| state | classic | refresh |
|---|---|---|
| default | ![Screenshot 2025-06-11 at 09 59 49](https://github.com/user-attachments/assets/f791bff9-6628-4d35-90ad-721de1d2441f) | ![Screenshot 2025-06-11 at 17 23 42](https://github.com/user-attachments/assets/3fdabf35-9332-4edd-bd5a-27bc73fe5290) | 
| file | ![Screenshot 2025-06-11 at 10 00 37](https://github.com/user-attachments/assets/8c203110-aa3e-4c56-97bc-15fe5dc3a613) | ![Screenshot 2025-06-11 at 17 23 51](https://github.com/user-attachments/assets/ebe98486-bf06-4abf-a2cb-d853e04a0200) | 
| hover | ![Screenshot 2025-06-11 at 10 00 06](https://github.com/user-attachments/assets/b2a981b3-c0ee-47c5-8f38-182f6ed60b1f) | ![Screenshot 2025-06-11 at 17 23 45](https://github.com/user-attachments/assets/fdc2b496-d88f-422d-82b1-27cfa637975c) | 
| focus | ![Screenshot 2025-06-11 at 10 00 17](https://github.com/user-attachments/assets/a6e3815e-cc68-44b7-89df-093f0df9503d) | ![Screenshot 2025-06-11 at 10 06 21](https://github.com/user-attachments/assets/5e16a5ee-0ff3-45ed-a79d-4f9afe53bf3f) | 
| dragover | ![Screenshot 2025-06-11 at 10 07 07](https://github.com/user-attachments/assets/ec403a91-8d8a-4a56-b709-ffab1a2591fe) | ![Screenshot 2025-06-11 at 10 14 00](https://github.com/user-attachments/assets/517e6dd9-6bff-46b6-87b2-fcbe39f6ba4b) | 
| invalid | ![Screenshot 2025-06-11 at 10 04 23](https://github.com/user-attachments/assets/b8ff85ff-e76d-4add-a39b-641e477257b9) | ![Screenshot 2025-06-11 at 10 09 11](https://github.com/user-attachments/assets/520641e5-9f21-4e67-81a1-1abcd34f5048) | 
| disabled | ![Screenshot 2025-06-11 at 10 04 19](https://github.com/user-attachments/assets/9cd46a62-a028-4607-92c0-ab8390ad9ffd) | ![Screenshot 2025-06-11 at 10 09 14](https://github.com/user-attachments/assets/735ebcea-3004-4444-aed9-aa7550803195) | 
| loading | ![Screenshot 2025-06-11 at 10 04 15](https://github.com/user-attachments/assets/ae5d81dd-99a1-4efd-a718-62cfa8147a99) | ![Screenshot 2025-06-11 at 17 23 38](https://github.com/user-attachments/assets/dfff5589-6d39-4c04-92e2-b18a11e41bde) | 


#### DARK mode

##### EuiComboBox

| classic | refresh |
|---|---|
| ![Screenshot 2025-06-11 at 13 25 19](https://github.com/user-attachments/assets/f3afe2f4-0426-47e3-b178-92e1295e4786) | ![Screenshot 2025-06-11 at 13 26 17](https://github.com/user-attachments/assets/18f38ce8-2ec1-4a0f-bfad-b107cca2ecfb) |
| ![Screenshot 2025-06-11 at 13 30 00](https://github.com/user-attachments/assets/378f530f-ae02-46d2-abcc-b692b30aa6cd) | ![Screenshot 2025-06-11 at 13 29 52](https://github.com/user-attachments/assets/aa0f0210-ad2c-41e6-9640-8eda23044f31) |  
| ![Screenshot 2025-06-11 at 13 29 57](https://github.com/user-attachments/assets/a347cd59-00ef-4661-9f92-560bee9786a8) | ![Screenshot 2025-06-11 at 13 29 48](https://github.com/user-attachments/assets/ce401679-464f-4418-901c-b8d090e01ff2) | 

##### EuiDatePicker

| classic | refresh |
|---|---|
| ![Screenshot 2025-06-11 at 09 29 30](https://github.com/user-attachments/assets/40d1fca9-07eb-4d05-80aa-8fe70f0a8eae) | ![Screenshot 2025-06-11 at 09 29 51](https://github.com/user-attachments/assets/b650662d-647f-4e68-922f-d59617abc9fd) |
| ![Screenshot 2025-06-11 at 09 31 23](https://github.com/user-attachments/assets/3cdd316c-964e-47db-9e31-62a72a2293c1) | ![Screenshot 2025-06-11 at 09 31 31](https://github.com/user-attachments/assets/ec1c5636-37ac-4f79-94c5-15daba9cbccc) |
| ![Screenshot 2025-06-11 at 09 33 57](https://github.com/user-attachments/assets/f7d136fc-cb17-4ecd-b5a7-1ab70be89bf5) | ![Screenshot 2025-06-11 at 09 34 10](https://github.com/user-attachments/assets/592110e7-a316-4f6f-aefd-08841bb8286f) | 

##### EuiSuperDatePicker

| classic | refresh |
|---|---|
| ![Screenshot 2025-06-11 at 09 38 49](https://github.com/user-attachments/assets/b3f66f21-d4e5-4104-a144-4d09ec84ef0d) | ![Screenshot 2025-06-11 at 09 40 29](https://github.com/user-attachments/assets/2ad886a4-9f24-4e9c-86ed-dc56b037c7b9) | 
| ![Screenshot 2025-06-11 at 09 49 09](https://github.com/user-attachments/assets/b9319f83-520e-46b1-a656-8c816f823172) | ![Screenshot 2025-06-11 at 09 49 03](https://github.com/user-attachments/assets/5fb5a8e0-4479-465c-9671-1bc8b7341265) | 
| ![Screenshot 2025-06-11 at 09 51 08](https://github.com/user-attachments/assets/95b56cc7-d299-4418-9110-7be4520450c2) | ![Screenshot 2025-06-11 at 09 56 30](https://github.com/user-attachments/assets/c616e204-fe92-4031-bc16-6f39972021ca) | 
| ![Screenshot 2025-06-11 at 09 52 13](https://github.com/user-attachments/assets/c6ffe3db-7549-4591-88c8-07defc7bb03d) | ![Screenshot 2025-06-11 at 09 53 10](https://github.com/user-attachments/assets/9f4da4ba-8fdb-403f-93a9-f50f6ca937cd) | 

##### EuiFilePicker

| state | classic | refresh |
|---|---|---|
| default | ![Screenshot 2025-06-11 at 10 01 01](https://github.com/user-attachments/assets/b8e40893-3972-4d99-bf97-2be2f3bcc106) | ![Screenshot 2025-06-11 at 17 22 27](https://github.com/user-attachments/assets/5100c85d-9cce-4562-8f68-ef73a725b10b) | 
| file | ![Screenshot 2025-06-11 at 10 00 45](https://github.com/user-attachments/assets/ca5a11b7-a0c2-403b-aed7-371eaa0665b1) | ![Screenshot 2025-06-11 at 17 22 23](https://github.com/user-attachments/assets/c50334d1-e98f-456b-86d2-c196d4b0aaa5) | 
| hover | ![Screenshot 2025-06-11 at 10 01 00](https://github.com/user-attachments/assets/1c731ac8-3a31-4304-a13f-afca4255b0cc) | ![Screenshot 2025-06-11 at 17 22 31](https://github.com/user-attachments/assets/029e36c7-04bb-4841-a012-1bea8d284886) | 
| focus | ![Screenshot 2025-06-11 at 10 00 52](https://github.com/user-attachments/assets/8243f53e-f8d5-487c-a138-c545b2d34d25) | ![Screenshot 2025-06-11 at 10 10 02](https://github.com/user-attachments/assets/b428c46b-c921-45a7-a4da-0249ea508df7) | 
| dragover | ![Screenshot 2025-06-11 at 10 07 16](https://github.com/user-attachments/assets/10adec20-8a74-4f5b-9af8-b057fb354433) | ![Screenshot 2025-06-11 at 10 12 58](https://github.com/user-attachments/assets/a1294331-112b-468c-9f53-2d0532f0f69f) | 
| invalid | ![Screenshot 2025-06-11 at 10 03 49](https://github.com/user-attachments/assets/9a85091c-d81a-4193-abd4-957c4fa19be8) | ![Screenshot 2025-06-11 at 10 10 27](https://github.com/user-attachments/assets/e65a2c99-b7e8-449e-aa01-8f6b21726188) | 
| disabled | ![Screenshot 2025-06-11 at 10 03 54](https://github.com/user-attachments/assets/c2da0be2-8aa0-43c1-8ee0-b8592a6d2404) | ![Screenshot 2025-06-11 at 10 10 29](https://github.com/user-attachments/assets/1776d485-76a6-41f1-88ff-14bd94866dcd) | 
| loading | ![Screenshot 2025-06-11 at 10 04 09](https://github.com/user-attachments/assets/de71e15f-85dd-4a60-a6c1-996cefb5c272) | ![Screenshot 2025-06-11 at 17 22 36](https://github.com/user-attachments/assets/73b39a82-e371-41b0-9386-3927fcc61d72) | 

#### High contrast mode

##### `preferred` HCM

###### EuiComboBox

| light | dark |
|---|---|
| ![Screenshot 2025-06-11 at 13 34 31](https://github.com/user-attachments/assets/79fae6a5-a1e8-4e07-b0fa-b965d80af6a6) | ![Screenshot 2025-06-11 at 13 34 12](https://github.com/user-attachments/assets/bf3d6308-08d9-4552-8f98-ae0565c57e8d) |

###### EuiDatePicker

| light | dark |
|---|---|
| ![Screenshot 2025-06-11 at 13 17 01](https://github.com/user-attachments/assets/7995fccf-24bc-4477-84a3-41e52a85eca4) | ![Screenshot 2025-06-11 at 13 16 49](https://github.com/user-attachments/assets/9937d6fc-14e7-486c-89cc-a2473c5addf7) |

###### EuiSuperDatePicker

| light | dark |
|---|---|
| ![Screenshot 2025-06-11 at 13 09 38](https://github.com/user-attachments/assets/b1820d16-13bd-41ce-8a98-b2e6e646a5cb) | ![Screenshot 2025-06-11 at 13 10 34](https://github.com/user-attachments/assets/cfb6e43a-ee06-45f6-95a3-e174dda53b68) |
| ![Screenshot 2025-06-11 at 13 09 54](https://github.com/user-attachments/assets/782257e6-047e-411a-84c8-0c4a47a39281) | ![Screenshot 2025-06-11 at 13 10 47](https://github.com/user-attachments/assets/12058a71-5bde-4882-84b3-8331169b396b) |
| ![Screenshot 2025-06-11 at 13 11 07](https://github.com/user-attachments/assets/d0482b80-1abf-4b40-8fbf-2f1cb8b44c46) | ![Screenshot 2025-06-11 at 13 11 02](https://github.com/user-attachments/assets/9d651c43-a4bd-48a2-9cb3-f9213f13fdf1) |

###### EuiFilePicker

| light | dark |
|---|---|
| ![Screenshot 2025-06-11 at 17 14 19](https://github.com/user-attachments/assets/32edb321-09f1-433c-a995-2dfee4760edc) | ![Screenshot 2025-06-11 at 17 14 39](https://github.com/user-attachments/assets/adc18d18-12eb-4757-82f9-cf1e165569f8) |

##### `forced` HCM

###### EuiComboBox

| light | dark |
|---|---|
| ![Screenshot 2025-06-11 at 13 35 51](https://github.com/user-attachments/assets/703bea8b-0833-455e-afa3-85454efe9030) | ![Screenshot 2025-06-11 at 13 35 35](https://github.com/user-attachments/assets/ff502e68-50bc-495c-9ebb-f8c01c086d6b) |

###### EuiDatePicker

| light | dark |
|---|---|
| ![Screenshot 2025-06-11 at 13 14 40](https://github.com/user-attachments/assets/da28cadc-7c14-40a7-a884-dbb550b99a49) | ![Screenshot 2025-06-11 at 13 14 48](https://github.com/user-attachments/assets/591540a6-ccef-49e1-b1a2-e0c56ce27b92) |

###### EuiSuperDatePicker

| light | dark |
|---|---|
| ![Screenshot 2025-06-11 at 13 08 26](https://github.com/user-attachments/assets/0784005b-72e6-4fa6-9685-6a6ad9135a9f) | ![Screenshot 2025-06-11 at 13 05 03](https://github.com/user-attachments/assets/d1225f16-4771-4402-b15a-1c0b9f6a5775) |
| ![Screenshot 2025-06-11 at 13 08 43](https://github.com/user-attachments/assets/36fe147e-5c9b-4474-829b-97448847e55a) | ![Screenshot 2025-06-11 at 13 04 31](https://github.com/user-attachments/assets/58fbb5e9-2c72-46f3-9612-2303d3be5f9c) |
| ![Screenshot 2025-06-11 at 13 12 33](https://github.com/user-attachments/assets/3dd45ccc-15f4-41b5-b3aa-9e879908fe83) | !![Screenshot 2025-06-11 at 13 12 56](https://github.com/user-attachments/assets/5e9dfe67-e1ca-4d38-85ee-d514b5932397) |

###### EuiFilePicker

| light | dark |
|---|---|
| ![Screenshot 2025-06-11 at 17 15 20](https://github.com/user-attachments/assets/88134ea3-37d1-4f24-a996-03feb7eb557f) | ![Screenshot 2025-06-11 at 17 15 10](https://github.com/user-attachments/assets/6f54335b-3ac6-41d8-b889-d7293cb024a0) |

</details>

## Impact to users

<!--
How will this change impact EUI users? If it's a breaking change, what will they need to do to handle this change when upgrading? Take a moment to look at usage in Kibana and consider how many usages this will impact and note it here.

Even if it is not a breaking change, how significant is the visual change? Is it a large enough visual change that we would want them advise them to test it?
-->

## QA

- [EuiComboBox](https://eui.elastic.co/pr_8778/docs/components/forms/selection/combo-box/)
- [EuiDatePicker](https://eui.elastic.co/pr_8778/docs/components/forms/date-and-time/date-picker/)
- [EuiSuperDatePicker](https://eui.elastic.co/pr_8778/docs/components/forms/date-and-time/super-date-picker/)
- [EuiFilePicker](https://eui.elastic.co/pr_8778/docs/components/forms/other/file-picker/) 

Review the directly affected components and verify:

- [ ] they matches the [design specs](https://www.figma.com/design/FHys7gLzyvD1gc9DrJM6D8/Elastic-UI--Borealis-?node-id=4422-2787) (sizes, colors, borders, states etc) for LIGHT and DARK mode
  - [ ] there are no unexpected visual issues in HCM
  - [ ] there are no regressions for Amsterdam

### General checklist

- Browser QA
    - [x] Checked in both **light and dark** modes
    - [x] Checked in both [MacOS](https://support.apple.com/lv-lv/guide/mac-help/unac089/mac) and [Windows](https://support.microsoft.com/en-us/windows/turn-high-contrast-mode-on-or-off-in-windows-909e9d89-a0f9-a3a9-b993-7a6dcee85025) **high contrast modes**
      - (_[emulate forced colors](https://devtoolstips.org/tips/en/emulate-forced-colors/) if you do not have access to a Windows machine_.)
    - [ ] ~Checked in **mobile**~
    - [x] Checked in **Chrome**, **Safari**, **Edge**, and **Firefox**
    - [ ] ~Checked for **accessibility** including keyboard-only and screenreader modes~
- Docs site QA
    - [ ] ~Added **[documentation](https://github.com/elastic/eui/blob/main/wiki/contributing-to-eui/documenting)**~
    - [ ] ~Props have proper **autodocs** (using `@default` if default values are missing) and **[playground toggles](https://github.com/elastic/eui/blob/main/wiki/contributing-to-eui/documenting/playgrounds.md)**~
    - [ ] ~Checked **[Code Sandbox](https://codesandbox.io/)** works for any docs examples~
- Code quality checklist
    - [x] Added or updated **[jest](https://github.com/elastic/eui/blob/main/wiki/contributing-to-eui/testing/unit-testing.md) and [cypress](https://github.com/elastic/eui/blob/main/wiki/contributing-to-eui/testing/cypress-testing.md) tests**
    - [x] Updated **[visual regression tests](https://github.com/elastic/eui/blob/main/wiki/contributing-to-eui/testing/visual-regression-testing.md)**
- Release checklist
    - [x] A **[changelog](https://github.com/elastic/eui/blob/main/wiki/contributing-to-eui/documenting/changelogs.md)** entry exists and is marked appropriately.
    - [ ] ~If applicable, added the **breaking change** issue label (and filled out the breaking change checklist)~
- Designer checklist
  - [ ] ~If applicable, [file an issue](https://github.com/elastic/platform-ux-team/issues/new/choose) to update [EUI's Figma library](https://www.figma.com/community/file/964536385682658129) with any corresponding UI changes. _(This is an internal repo, if you are external to Elastic, ask a maintainer to submit this request)_~

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4155)
* Commit: 18bcbf521dceec0cc98b0b9b18be0deaebc84e32

cc @mgadewoll

<!--buildkite-pr-comment-eui-pull-request-test-and-deploy
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4155","number":4155,"commit":"18bcbf521dceec0cc98b0b9b18be0deaebc84e32"}],"number":4155}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/eui-deploy-docs/builds/494)
* Commit: bbc6c3b3165a2c82f3cdfef6eeae302bb9177bc3
* [Documentation website](https://eui.elastic.co/pr_8778/)


### History
* :green_heart: [Build #480](https://buildkite.com/elastic/eui-deploy-docs/builds/480) succeeded 18bcbf521dceec0cc98b0b9b18be0deaebc84e32

cc @mgadewoll

<!--buildkite-pr-comment-eui-deploy-docs
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/494","number":494,"commit":"bbc6c3b3165a2c82f3cdfef6eeae302bb9177bc3"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/480","number":480,"commit":"18bcbf521dceec0cc98b0b9b18be0deaebc84e32"}],"number":494}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4164)
* Commit: bbc6c3b3165a2c82f3cdfef6eeae302bb9177bc3

### History
* :green_heart: [Build #4155](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4155) succeeded 18bcbf521dceec0cc98b0b9b18be0deaebc84e32

cc @mgadewoll

<!--buildkite-pr-comment-eui-pull-request-test-and-deploy
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4164","number":4164,"commit":"bbc6c3b3165a2c82f3cdfef6eeae302bb9177bc3"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4155","number":4155,"commit":"18bcbf521dceec0cc98b0b9b18be0deaebc84e32"}],"number":4164}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/eui-deploy-docs/builds/496)
* Commit: cd72213473fa6285c13a42a6d0cede52dacd047a
* [Documentation website](https://eui.elastic.co/pr_8778/)


### History
* :green_heart: [Build #494](https://buildkite.com/elastic/eui-deploy-docs/builds/494) succeeded bbc6c3b3165a2c82f3cdfef6eeae302bb9177bc3
* :green_heart: [Build #480](https://buildkite.com/elastic/eui-deploy-docs/builds/480) succeeded 18bcbf521dceec0cc98b0b9b18be0deaebc84e32

cc @mgadewoll

<!--buildkite-pr-comment-eui-deploy-docs
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/496","number":496,"commit":"cd72213473fa6285c13a42a6d0cede52dacd047a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/494","number":494,"commit":"bbc6c3b3165a2c82f3cdfef6eeae302bb9177bc3"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/480","number":480,"commit":"18bcbf521dceec0cc98b0b9b18be0deaebc84e32"}],"number":496}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4216)
* Commit: 7bf8354429ae40db4f0adea596e5e15a87e3d4a7

### History
* :green_heart: [Build #4165](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4165) succeeded cd72213473fa6285c13a42a6d0cede52dacd047a
* :green_heart: [Build #4164](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4164) succeeded bbc6c3b3165a2c82f3cdfef6eeae302bb9177bc3
* :green_heart: [Build #4155](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4155) succeeded 18bcbf521dceec0cc98b0b9b18be0deaebc84e32

cc @mgadewoll

<!--buildkite-pr-comment-eui-pull-request-test-and-deploy
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4216","number":4216,"commit":"7bf8354429ae40db4f0adea596e5e15a87e3d4a7"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4165","number":4165,"commit":"cd72213473fa6285c13a42a6d0cede52dacd047a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4164","number":4164,"commit":"bbc6c3b3165a2c82f3cdfef6eeae302bb9177bc3"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4155","number":4155,"commit":"18bcbf521dceec0cc98b0b9b18be0deaebc84e32"}],"number":4216}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/eui-deploy-docs/builds/562)
* Commit: 3066875aeaa62b32101b55cf684634c36fd6cab0
* [Documentation website](https://eui.elastic.co/pr_8778/)


### History
* :green_heart: [Build #559](https://buildkite.com/elastic/eui-deploy-docs/builds/559) succeeded 19cc0ed600b8e1d7ec1f764e304635ff79daf30b
* :green_heart: [Build #555](https://buildkite.com/elastic/eui-deploy-docs/builds/555) succeeded 7bf8354429ae40db4f0adea596e5e15a87e3d4a7
* :green_heart: [Build #496](https://buildkite.com/elastic/eui-deploy-docs/builds/496) succeeded cd72213473fa6285c13a42a6d0cede52dacd047a
* :green_heart: [Build #494](https://buildkite.com/elastic/eui-deploy-docs/builds/494) succeeded bbc6c3b3165a2c82f3cdfef6eeae302bb9177bc3
* :green_heart: [Build #480](https://buildkite.com/elastic/eui-deploy-docs/builds/480) succeeded 18bcbf521dceec0cc98b0b9b18be0deaebc84e32

cc @mgadewoll

<!--buildkite-pr-comment-eui-deploy-docs
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/562","number":562,"commit":"3066875aeaa62b32101b55cf684634c36fd6cab0"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/559","number":559,"commit":"19cc0ed600b8e1d7ec1f764e304635ff79daf30b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/555","number":555,"commit":"7bf8354429ae40db4f0adea596e5e15a87e3d4a7"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/496","number":496,"commit":"cd72213473fa6285c13a42a6d0cede52dacd047a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/494","number":494,"commit":"bbc6c3b3165a2c82f3cdfef6eeae302bb9177bc3"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/480","number":480,"commit":"18bcbf521dceec0cc98b0b9b18be0deaebc84e32"}],"number":562}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4220)
* Commit: 3066875aeaa62b32101b55cf684634c36fd6cab0

### History
* :green_heart: [Build #4218](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4218) succeeded 19cc0ed600b8e1d7ec1f764e304635ff79daf30b
* :green_heart: [Build #4216](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4216) succeeded 7bf8354429ae40db4f0adea596e5e15a87e3d4a7
* :green_heart: [Build #4165](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4165) succeeded cd72213473fa6285c13a42a6d0cede52dacd047a
* :green_heart: [Build #4164](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4164) succeeded bbc6c3b3165a2c82f3cdfef6eeae302bb9177bc3
* :green_heart: [Build #4155](https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4155) succeeded 18bcbf521dceec0cc98b0b9b18be0deaebc84e32

cc @mgadewoll

<!--buildkite-pr-comment-eui-pull-request-test-and-deploy
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4220","number":4220,"commit":"3066875aeaa62b32101b55cf684634c36fd6cab0"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4218","number":4218,"commit":"19cc0ed600b8e1d7ec1f764e304635ff79daf30b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4216","number":4216,"commit":"7bf8354429ae40db4f0adea596e5e15a87e3d4a7"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4165","number":4165,"commit":"cd72213473fa6285c13a42a6d0cede52dacd047a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4164","number":4164,"commit":"bbc6c3b3165a2c82f3cdfef6eeae302bb9177bc3"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-pull-request-test-and-deploy/builds/4155","number":4155,"commit":"18bcbf521dceec0cc98b0b9b18be0deaebc84e32"}],"number":4220}
buildkite-pr-comment-->

acstll commented on packages/eui/src/components/form/file_picker/file_picker.styles.ts:
```diff
@@ -97,8 +155,14 @@ export const euiFilePickerStyles = (euiThemeContext: UseEuiTheme) => {
       color: ${euiTheme.colors.textParagraph};
-      border: ${euiTheme.border.width.thick} dashed
-        var(--euiFormControlStateColor, ${euiTheme.border.color});
+      border: ${euiTheme.border.width.thick}
```
```suggestion
      border: ${euiTheme.border.width.thin}
```

acstll commented on packages/eui/src/components/form/file_picker/file_picker.tsx:
```diff
@@ -143,184 +145,209 @@ export class EuiFilePickerClass extends Component<
+                      css={iconStyles}
+                      className="euiFilePicker__icon"
+                      color={
+                        isInvalid
+                          ? 'danger'
+                          : disabled
+                          ? isRefreshVariant
+                            ? 'disabled'
+                            : 'subdued'
+                          : isRefreshVariant
+                          ? 'text'
+                          : 'primary'
```
[non-blocking] this looks hard to read — I have no actual suggestion though 😛

acstll replied:
yep, I would keep it as it is, maybe add a comment

---

_my friend_ recommends this:

```ts
function getIconColor(isInvalid: boolean, disabled: boolean, isRefreshVariant: boolean): string {
  if (isInvalid) {
    return 'danger';
  }

  if (disabled) {
    return isRefreshVariant ? 'disabled' : 'subdued';
  }

  if (isRefreshVariant) {
    return 'text';
  }

  return 'primary';
}
```

or this:

```ts
function getIconColor(isInvalid: boolean, disabled: boolean, isRefreshVariant: boolean): string {
  switch (true) {
    case isInvalid:
      return 'danger';
    case disabled && isRefreshVariant:
      return 'disabled';
    case disabled:
      return 'subdued';
    case isRefreshVariant:
      return 'text';
    default:
      return 'primary';
  }
}
```

weronikaolejniczak replied:
@acstll is your friend's name AI? 😄

acstll reviewed (commented):
Same review process as in PR number 1. Only 1 comment affecting the code, everything else is looking good!

Found a couple of small things, all related to `EuiFilePicker`.

### matches the [design specs](https://www.figma.com/design/FHys7gLzyvD1gc9DrJM6D8/Elastic-UI--Borealis-?node-id=4422-2787)

- `EuiFilePicker`
  - `border-width` in the spec is 1px (fix suggested)
  - [no spec?] the color for the animated line in the loading state has a red color that suggests "invalid" (in Amsterdam looks more magenta-ish)
  - [maybe out of scope] the size of the "remove" button is different, I believe they're both `xs` in spec and in code (not sure what's happening)
  - [nit, feel free to ignore] on hover, the icon grows 1px, in code it scales 1.1 which seems more, I would actually remove this animation since it reminds me of Amsterdam
  
<details>
<summary>Toggle screenshots</summary>

#### loading-state line color
![Screenshot 2025-06-20 at 10 33 48](https://github.com/user-attachments/assets/775a981a-2ecf-4f8d-8b04-7eb14dc8ef54)

#### "remove" button size
![Screenshot 2025-06-20 at 10 43 54](https://github.com/user-attachments/assets/5be424a3-81c7-442e-97ca-27d2b6229de1)
</details>

### no unexpected visual issues in HCM

looking good 👌 

### no regressions for Amsterdam

- `EuiFilePicker` in HCM, the border on hover is solid, not dashed — I 
wonder if this may be intended?

mgadewoll commented:
> the color for the animated line in the loading state has a red color that suggests "invalid" (in Amsterdam looks more magenta-ish)

@acstll Correct, there is no spec for it which is why I left it as is. Maybe it could rather be primary similar to loading spinners? 🤔  That would be a question for @ek-so if the color could be updated in Borealis.

> [maybe out of scope] the size of the "remove" button is different, I believe they're both xs in spec and in code (not sure what's happening)

I checked and actually the `xs` empty button doesn't match the design specs. Looks like we missed that in the Button update. 🫠 
I'll have to check if that impacts too many other places that would require it to be a separate update.

> [nit, feel free to ignore] on hover, the icon grows 1px, in code it scales 1.1 which seems more, I would actually remove this animation since it reminds me of Amsterdam

Removing it would be a suggestion to design. Given that design specs provide the size change, I'd leave it here for now but I can adjust the `scale()`. I left it at `1.1` to not have to go to even smaller decimals but it won't hurt anyway.
Updated in https://github.com/elastic/eui/pull/8778/commits/8c8c02436e5ab7a4bf0625613ce6f021e88dc1d8

>EuiFilePicker in HCM, the border on hover is solid, not dashed — I
wonder if this may be intended?

That was intentional from my side (sorry for not mentioning it explicitly). The idea being to improve the visibility. But thinking about it again now, it might not be needed/wanted considering other form elements don't behave that way either. I'll revert it. 👍  
Updated in https://github.com/elastic/eui/pull/8778/commits/8c8c02436e5ab7a4bf0625613ce6f021e88dc1d8

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/eui-deploy-docs/builds/592)
* Commit: 8c8c02436e5ab7a4bf0625613ce6f021e88dc1d8
* [Documentation website](https://eui.elastic.co/pr_8778/)


### History
* :green_heart: [Build #562](https://buildkite.com/elastic/eui-deploy-docs/builds/562) succeeded 3066875aeaa62b32101b55cf684634c36fd6cab0
* :green_heart: [Build #559](https://buildkite.com/elastic/eui-deploy-docs/builds/559) succeeded 19cc0ed600b8e1d7ec1f764e304635ff79daf30b
* :green_heart: [Build #555](https://buildkite.com/elastic/eui-deploy-docs/builds/555) succeeded 7bf8354429ae40db4f0adea596e5e15a87e3d4a7
* :green_heart: [Build #496](https://buildkite.com/elastic/eui-deploy-docs/builds/496) succeeded cd72213473fa6285c13a42a6d0cede52dacd047a
* :green_heart: [Build #494](https://buildkite.com/elastic/eui-deploy-docs/builds/494) succeeded bbc6c3b3165a2c82f3cdfef6eeae302bb9177bc3
* :green_heart: [Build #480](https://buildkite.com/elastic/eui-deploy-docs/builds/480) succeeded 18bcbf521dceec0cc98b0b9b18be0deaebc84e32

cc @mgadewoll

<!--buildkite-pr-comment-eui-deploy-docs
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/592","number":592,"commit":"8c8c02436e5ab7a4bf0625613ce6f021e88dc1d8"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/562","number":562,"commit":"3066875aeaa62b32101b55cf684634c36fd6cab0"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/559","number":559,"commit":"19cc0ed600b8e1d7ec1f764e304635ff79daf30b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/555","number":555,"commit":"7bf8354429ae40db4f0adea596e5e15a87e3d4a7"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/496","number":496,"commit":"cd72213473fa6285c13a42a6d0cede52dacd047a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/494","number":494,"commit":"bbc6c3b3165a2c82f3cdfef6eeae302bb9177bc3"}],"number":592}
buildkite-pr-comment-->

mgadewoll commented:
ℹ️ Updated the button `xs` font-size ([commit](https://github.com/elastic/eui/pull/8778/commits/af31d01b85b30891328383ba4693fe53817585ba)). The changes triggered a couple of VRT image updates. Those changes are expected and aligned with design.

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/eui-deploy-docs/builds/595)
* Commit: 5dde0f736ca9d27fd527557cbce1371be11afed7
* [Documentation website](https://eui.elastic.co/pr_8778/)


### History
* :green_heart: [Build #592](https://buildkite.com/elastic/eui-deploy-docs/builds/592) succeeded 8c8c02436e5ab7a4bf0625613ce6f021e88dc1d8
* :green_heart: [Build #562](https://buildkite.com/elastic/eui-deploy-docs/builds/562) succeeded 3066875aeaa62b32101b55cf684634c36fd6cab0
* :green_heart: [Build #559](https://buildkite.com/elastic/eui-deploy-docs/builds/559) succeeded 19cc0ed600b8e1d7ec1f764e304635ff79daf30b
* :green_heart: [Build #555](https://buildkite.com/elastic/eui-deploy-docs/builds/555) succeeded 7bf8354429ae40db4f0adea596e5e15a87e3d4a7
* :green_heart: [Build #496](https://buildkite.com/elastic/eui-deploy-docs/builds/496) succeeded cd72213473fa6285c13a42a6d0cede52dacd047a
* :green_heart: [Build #494](https://buildkite.com/elastic/eui-deploy-docs/builds/494) succeeded bbc6c3b3165a2c82f3cdfef6eeae302bb9177bc3

cc @mgadewoll

<!--buildkite-pr-comment-eui-deploy-docs
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/595","number":595,"commit":"5dde0f736ca9d27fd527557cbce1371be11afed7"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/592","number":592,"commit":"8c8c02436e5ab7a4bf0625613ce6f021e88dc1d8"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/562","number":562,"commit":"3066875aeaa62b32101b55cf684634c36fd6cab0"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/559","number":559,"commit":"19cc0ed600b8e1d7ec1f764e304635ff79daf30b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/555","number":555,"commit":"7bf8354429ae40db4f0adea596e5e15a87e3d4a7"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/eui-deploy-docs/builds/496","number":496,"commit":"cd72213473fa6285c13a42a6d0cede52dacd047a"}],"number":595}
buildkite-pr-comment-->
